packages/utils/src/lib/logger.int.test.ts

```diff
@@ -267,14 +268,16 @@ ${ansis.cyan('└')} ${ansis.green(`Total line coverage is ${ansis.bold('82%')}`
     it('should use collapsible sections in GitLab CI/CD environment, initial collapse depends on verbosity', async () => {
       vi.stubEnv('CI', 'true');
       vi.stubEnv('GITLAB_CI', 'true');
-      vi.setSystemTime(new Date(123456789000)); // current Unix timestamp: 123456789 seconds since epoch
+      vi.setSystemTime(new Date(123_456_789_000)); // current Unix timestamp: 123456789 seconds since epoch
       performanceNowSpy
         .mockReturnValueOnce(0)
         .mockReturnValueOnce(123) // 1st group duration: 123 ms
         .mockReturnValueOnce(0)
         .mockReturnValue(45); // 2nd group duration: 45 ms
       mathRandomSpy
+        // eslint-disable-next-line unicorn/number-literal-case
         .mockReturnValueOnce(0x1a / Math.pow(2, 8)) // 1st group's random ID: "1a"
+        // eslint-disable-next-line unicorn/number-literal-case
         .mockReturnValueOnce(0x1b / Math.pow(2, 8)); // 2nd group's random ID: "1b"
```

Comment:
I'd autofix the number formatting. I don't really mind whether the hexadecimals are in uppercase or lowercase, so I'd stick to the convention.

```suggestion
        .mockReturnValueOnce(0x1A / Math.pow(2, 8)) // 1st group's random ID: "1a"
        .mockReturnValueOnce(0x1B / Math.pow(2, 8)); // 2nd group's random ID: "1b"
```

Reply:
There is a conflict between Prettier and ESLint. Prettier formats hexadecimal literals to lowercase, whereas ESLint flags them as errors for not being uppercase. It seems one should be disabled.